Makefile

```diff
@@ -160,10 +160,11 @@ build-dev: build-ci
 ################################################################################
 # Target: build-webui
 ################################################################################
-WEB_GO_FILES := $(shell find webui -name '*.go')
+EB_GO_FILES := $(shell find webui -name '*.go')
```

Comment:
Looks like it's missing the initial W, where WEB_GO_FILES is used by the BINARY_PATH target in this Makefile.